src/pages/landing/landing.css

```
@@ -0,0 +1,749 @@
+/* ============================================================
+   WhyLog Landing page — scoped styles
+   Ported from the design bundle (WhyLog Landing.html).
+   ============================================================ */
+
+.whylog-landing {
+  font-family: "Pretendard", "Pretendard Variable", "Inter", sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)